catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraItems.scala

```diff
@@ -0,0 +1,55 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import com.twitter.inject.Logging
+import scala.util.{Failure, Success}
+import uk.ac.wellcome.models.{
+  IdentifierSchemes,
+  SourceIdentifier,
+  UnidentifiedItem
+}
+import uk.ac.wellcome.models.transformable.SierraTransformable
+import uk.ac.wellcome.transformer.source.SierraItemData
+import uk.ac.wellcome.utils.JsonUtil._
+
+trait SierraItems extends Logging with SierraLocation {
+  def extractItemData(
+    sierraTransformable: SierraTransformable): List[SierraItemData] = {
```

Comment:
I might put some of this logic on a companion object for `SierraItemData` in an `apply` method.